tools/mpremote/mpremote/main.py

```diff
@@ -212,6 +222,10 @@ def argparse_none(description):
         do_connect,
         argparse_connect,
     ),
+    "delay": (
```

Comment:
maybe this should be `delay_ms` to be clear on the units (and match `sleep_ms`)?

Reply:
I went with millis to match the existing args to reset/bootloader, but the new sleep command kind of makes those arguments unnecessary, so removed them instead. `sleep` now uses seconds (to match unix `sleep(1)`).